src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search"
+    action="index.html"
+    method="post"
+    data-qa="big">
```

Comment:
Need to fix code styles everywhere
```suggestion
    <form
      class="search"
      action="index.html"
      method="post"
      data-qa="big"
    >
```